catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -73,18 +77,27 @@
 import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
-import org.awaitility.Awaitility;
-import org.awaitility.core.ConditionTimeoutException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class HadoopCatalogOperations extends ManagedSchemaOperations
     implements CatalogOperations, FilesetCatalog {
   private static final String SCHEMA_DOES_NOT_EXIST_MSG = "Schema %s does not exist";
   private static final String FILESET_DOES_NOT_EXIST_MSG = "Fileset %s does not exist";
+  private static final ThreadPoolExecutor FILE_SYSTEM_EXECUTOR =
+      new ThreadPoolExecutor(
+          2,
+          Math.max(2, Runtime.getRuntime().availableProcessors() / 2),
```

Comment:
Please make the thread number alias with the default value as the jetty server, see 
https://github.com/apache/gravitino/blob/261562538c008b2395079789a6bfbfc625bb39c6/server-common/src/main/java/org/apache/gravitino/server/web/JettyServerConfig.java#L63-L78